tests/functional/verification/test_reflected_,magic_func.py

```diff
@@ -0,0 +1,202 @@
+from nagini_contracts.contracts import *
```

Comment:
The file name looks weird, is there a comma in there??

Reply:
Yeah, I must have missed that typo. Fixed.